op-e2e/Makefile

```diff
@@ -13,6 +13,12 @@ test: pre-test
 lint:
 	golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint -e "errors.As" -e "errors.Is"
 
+test-external-%: pre-test external_%/shim
+	go test -v --externalL2 ./external_$*/shim
+
+external_%/shim: external_%/*.go
+	go build -o external_$*/shim ./external_$*/
```

Comment:
We probably shouldn't assume the external shim is written in go.  I'd just have this execute `make` in the shim's directory, then it can control how it builds and build any dependencies etc as well.